src/shared/components/accounts-table/accounts-table.component.html

```diff
@@ -0,0 +1,59 @@
+<table class="accounts-table">
+  <thead>
+    <tr>
+      <th>
+        <input type="checkbox" />
+      </th>
+      <th>
+        Имейл
+        <img src="../../../../assets/icons/table-arrows.png" alt="sort icon" />
```

Comment:
why such "icon" with image? There are icons for this.

Reply:
done